packages/ui-proxy/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "ui-proxy",
+  "version": "1.0.0",
+  "description": "Proxy server for Genestack frontend development",
```

Comment:
Since this package is for Node on some developer's machine, I think we need something like:
```json
"engines" : { "node" : ">=6.0.0 }
```
since we use ES2015 here.